swagger.yaml

```diff
@@ -585,19 +634,64 @@ definitions:
     properties:
       email:
         type: string
-        description: "Email of authorized user."
+        description: "Email of an authorized user."
       key_name:
         type: string
-        description: "Name of secret's key."
+        description: "Name of the secret's key."
   VersionListLinkRequestParameters:
     type: object
     properties:
       email:
         type: string
-        description: "Email of authorized user."
+        description: "Email of an authorized user."
       key_name:
         type: string
-        description: "Name of secret's key."
+        description: "Name of the secret's key."
+  UpdateClassificationRequestParameters:
+    type: object
+    properties:
+      email:
+        type: string
+        description: Email of an authorized user.
+      key_name: 
+        type: string
+        description: Name of the secret's key.
+      Classification: 
+        type: string
+        description:
```

Comment:
here is where you need to add something @CryptoMoveKDD, then I think we're good.